specs-go/config.go

```diff
@@ -79,6 +85,8 @@ type Platform struct {
 	OS string `json:"os"`
 	// Arch is the architecture
 	Arch string `json:"arch"`
+	// OSVersion is the version of the operating system. (this field is platform dependent)
+	OSVersion string `json:"os.version,omitempty" platform:"windows"`
```

Comment:
On Wed, Jun 22, 2016 at 07:55:51PM -0700, Mrunal Patel wrote:
> > +	// OSVersion is the version of the operating system. (this field is platform dependent)
> > +	OSVersion string `json:"os.version,omitempty" platform:"windows"`
> 
> Curious why can't we use the existing `OS` field?

Would you use ‘old-windows’ or some such for “Windows before build
14350” [1]?  I think it's better to keep ‘windows’ there and pass that
build information through as a separate thing.  I just think the
separate thing should be a runtime flag instead of a config option.

[1]: https://github.com/docker/docker/blob/v1.12.0-rc2/libcontainerd/process_windows.go#L34-L51